index.html

```diff
@@ -30,8 +33,15 @@ <h1 class="display-3">Work Day Scheduler</h1>
     </header>
     <div class="container">
       <!-- Timeblocks go here -->
+      <h1 class="text-center">Day Planner</h1>
```

Comment:
Start by creating time blocks for all hours of the day. Then from there create a save button for each block. After that find a way to save the text that is inputted in that textblock. Hint: <textarea> is a cool tag that can be saved using its value in local storage.